commands/shop.js

```diff
@@ -61,6 +63,47 @@ if(area[0]==4){
   }
   let choice = parseInt(args[0],10);
   if(isNaN(choice)){
+	if(!args[1]){
+		message.channel.send("That is not a valid shop item!");
+		return;
+	}
+	switch(args[0].toLowerCase())
+	{
+		case "price":
+			choice = parseInt(args[1],10);
+			if(isNaN(choice)){
+				message.channel.send("That is not a valid shop item!");
+				return;
+			}
+			if(choice<=0||choice>dex.length){
+				message.channel.send(`There are only ${dex.length} things to buy here! choose one of them!`);
+				return;
+			}
+			let thisPrice = client.charcall.charData(client,shopkeepId,"shopPrices")[choice-1];
+			message.channel.send(`That ${dex[choice-1][0]} costs ${thisPrice} BOONDOLLARS.`);
+			return;
+		case "inspect":
+			choice = parseInt(args[1],10);
+			if(isNaN(choice)){
+				message.channel.send("That is not a valid shop item!");
+				return;
+			}
+			if(choice<=0||choice>dex.length){
+				message.channel.send(`There are only ${dex.length} things to buy here! choose one of them!`);
+				return;
+			}
+			
+			async function itemInspect()
+			{
+				const attachment = await client.imgcall.inspect(client,message,args,3,dex[choice-1]);
+				message.channel.send({content: "Inspecting item", files: [attachment]});
+			}
+			itemInspect()
+			return;
+		default:
+			message.channel.send("That is not a valid shop item!");
+			return;
+	}
     message.channel.send("That is not a valid shop item!");
```

Comment:
Lines 104 and 105 will never be used, because even the default case in the switch over them has a return.

Reply:
You make a good point. There's some other copy-pasted code in this switch statement I can probably do something about....